layouts/docs/baseof.html

```diff
@@ -24,6 +24,9 @@
             {{ if not .Site.Params.ui.breadcrumb_disable }}{{ partial "breadcrumb.html" . }}{{ end }}
             {{ block "deprecated" . }}
               {{ partial "deprecation-warning.html" . }}
+            {{ end }}
+            {{ block "outdated_content" . }}
```

Comment:
:thought_balloon: If a site is deprecated, maybe we don't want to also mention that the localization is stale (it might be).

Reply:
If the site is deprecated, it's possible to hide this message.
Would you like us to proceed with that?

Reply:
> If the site is deprecated, it's possible to hide this message.
> Would you like us to proceed with that?

I recommend it.

Reply:
fixed it!